biothings/hub/__init__.py

```diff
@@ -104,7 +104,7 @@ def _config_for_app(config_mod=None):
 
 def get_loop(max_workers=None):
     loop = asyncio.get_event_loop()
-    executor = concurrent.futures.ProcessPoolExecutor(max_workers=max_workers)
+    executor = concurrent.futures.ThreadPoolExecutor(max_workers=max_workers)
```

Comment:
With this change to `ThreadPoolExecutor`, did you test it works as expected?

Reply:
Yes it works. atm no where call this method
So I did a test by doing this method: in `biothings/hub/__init__.py::configure_job_manager`, when create a `JobManager` instance, I passed in `get_loop(max_workers)`, then retest the flow, and I saw the code still work.